sdl_android/src/main/java/com/smartdevicelink/api/SdlManager.java

```diff
@@ -636,6 +653,16 @@ public boolean removeOnRPCNotificationListener(FunctionID notificationId, OnRPCN
 			return proxy.removeOnRPCNotificationListener(notificationId,listener);
 		}
 
+		@Override
+		public void addOnRPCResponseListener(final FunctionID responseId, final OnRPCResponseListener listener) {
```

Comment:
This doesn't make sense. The response listener is specific to a correlation id sent by the request. This should only be `addOnRPCListener` and take a `OnRPCListener`. The remove method should be modified accordingly.